MapleServer2/PacketHandlers/Game/JobHandler.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.Extensions.Logging;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Packets;
+using MapleServer2.Types;
+
+namespace MapleServer2.PacketHandlers.Game {
+    public class JobHandler : GamePacketHandler {
+        public override ushort OpCode => RecvOp.JOB;
+
+        public JobHandler(ILogger<JobHandler> logger) : base(logger) { }
+
+        public override void Handle(GameSession session, PacketReader packet) {
+            byte mode = packet.ReadByte();
+            switch (mode) {
+                case 8: // Close Skill Tree
+                    HandleCloseSkillTree(session, packet);
+                    break;
+                case 9: // Save Skill Tree
+                    HandleSaveSkillTree(session, packet);
+                    break;
+            }
+        }
+
+        public void HandleCloseSkillTree(GameSession session, PacketReader packet) {
```

Comment:
This and the below method could be private

Reply:
Agreed.